### PR TITLE
Adds xp tables

### PR DESCRIPTION
**Pull request by Rotorsoft**
Create new xp projection model to track points

## Link to Issue
Closes: #9342

## Description of Changes
- Adds xp logs table
- Adds xp_points col to users

## Test Plan
- N/A

## Deployment Plan

1. 

## Other Considerations

-

### CODE REVIEW

File: libs/model/src/models/xp_log.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export type XpLogInstance = ModelInstance<z.infer<typeof XpLog>>;
+export type XpLogModelStatic = Sequelize.ModelStatic<XpLogInstance>;
+
+export default (sequelize: Sequelize.Sequelize) =>
+  <XpLogModelStatic>sequelize.define<XpLogInstance>(
+    'XpLog',
+    {
+      user_id: { type: Sequelize.INTEGER, primaryKey: true },
+      created_at: { type: Sequelize.DATE, primaryKey: true },
```
Comment (timolegros) [inline libs/model/src/models/xp_log.ts:14]: Is this value going to be managed by us?

**Review by timolegros — APPROVED** [review]

File: libs/model/src/models/xp_log.ts (hunk 1 above)
Comment (Rotorsoft) [reply libs/model/src/models/xp_log.ts:14]: yes, we don't need updated_at here, this is append only

Comment (timolegros) [reply libs/model/src/models/xp_log.ts:14]: Right but what I mean is, do we need to be providing the date manually for any reason or can we just have Sequelize manage it (and disable `updated_at`).

Comment (Rotorsoft) [reply libs/model/src/models/xp_log.ts:14]: didn't know you can only enable created_at with sequelize... can you show me one example and I'll adjust it

Comment (timolegros) [reply libs/model/src/models/xp_log.ts:14]: image
```
createdAt: 'created_at`,
updatedAt: false,
```

**Review by rbennettcw — APPROVED** [review]

**Pull request merged**